components/Map.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import ReactMapGL, { Marker } from "react-map-gl";
+export default function Map({ locations }) {
+  // UseState hook
+  const [selectedLocation, setSelectedLocation] = useState({});
+  const [viewport, setViewport] = useState({
+    width: "100%",
+    height: "100%",
+    // The latitude and longitude of the center of London
```

Comment:
What is the reason to use London instead use Cuenca!

Reply:
Already corrected the error, i taken a code of internet and have not change some things.